lnwallet/channel.go

```diff
@@ -3092,6 +3277,96 @@ func (lc *LightningChannel) createCommitDiff(
 	}, nil
 }
 
+// getUnsignedAckedUpdates returns all remote log updates that we haven't
+// signed for yet ourselves.
+func (lc *LightningChannel) getUnsignedAckedUpdates() []channeldb.LogUpdate {
+	// First, we need to convert the funding outpoint into the ID that's
+	// used on the wire to identify this channel.
+	chanID := lnwire.NewChanIDFromOutPoint(&lc.channelState.FundingOutpoint)
+
+	// Fetch the last remote update that we have signed for.
+	lastRemoteCommitted := lc.remoteCommitChain.tip().theirMessageIndex
```

Comment:
👍 w.r.t the slight reshuffling of logic here, I find it much easier to follow now.